tests/components/moonraker/const.py

```diff
@@ -0,0 +1,3 @@
+"""Constants for Moonraker integration."""
+
+DOMAIN = "moonraker"
```

Comment:
This isn't needed? The integration already has a const file that can be used/imported in tests. Let's remove this file.